apps/video/test/marionette/lib/video.js

```diff
@@ -130,14 +133,30 @@ Video.prototype = {
    * @return {Marionette.Element} First element of all thumbnail images.
    */
   get thumbnail() {
-    return this.client.findElement(Video.Selector.thumbnail);
+    return this.client.helper.waitForElement(Video.Selector.thumbnail);
   },
 
   /**
    * @return {Marionette.Element} video player element.
    */
   get player() {
-    return this.client.findElement(Video.Selector.player);
+    return this.client.helper.waitForElement(Video.Selector.player);
+  },
+
+  showControls: function() {
```

Comment:
I wonder about this method name. If you call it a second time it will actually hide the controls, right?  I think you should either make this something like ensureControlsShowing() . Or you should just get rid of the method and just call app.player.tap() directly in the test.